core/src/main/scala/magnolia1/magnolia.scala

```diff
@@ -258,9 +258,18 @@ object Magnolia {
 
     def extractParameterBlockFor(termName: String, category: String): List[Symbol] = {
       val term = TermName(termName)
+      val searchType = stack.top.map(_.searchType.toString)
+      val additionalInfo = searchType match {
+        case Some(tpe) => s"unable to derive $tpe -- "
+        case None      => ""
+      }
```

Comment:
This should move inside `getOrElse` to avoid doing work on the happy path.

Reply:
[e036de4](https://github.com/softwaremill/magnolia/pull/501/commits/e036de4a74e7ad2c829bb99e4131a54de4490dc0)
Thanks for the review.